chemprop/v2/tests/unit/data/test_dataset.py

```diff
@@ -96,13 +96,12 @@ def test_num_tasks(dataset, targets):
 
 
 def test_aux_nones(dataset: MoleculeDataset):
-    assert np.isnan(dataset.X_f).all()
-    # assert dataset.X_phase is None # there's no attribute in dataset that is relevant to phase?
-    np.testing.assert_array_equal(dataset.X_f, np.nan)  # .all()
-    np.testing.assert_array_equal(dataset.V_fs, np.nan)  # .all()
-    np.testing.assert_array_equal(dataset.E_fs, np.nan)  # .all()
-    np.testing.assert_array_equal(dataset.gt_mask, None)  # .all()
-    np.testing.assert_array_equal(dataset.lt_mask, None)  # .all()
+    np.testing.assert_array_equal(dataset.X_f, None)
+    np.testing.assert_array_equal(dataset.X_f, None)
```

Comment:
Duplicate test

Reply:
By the way, is this right? I think `dataset.X_f` should be a numpy array with `None` elements, but not `None` itself. See [here](https://github.com/chemprop/chemprop/blob/v2/dev/chemprop/v2/data/datasets.py#L56). The same go with other attributes?

Reply:
this asserts that all entries are `None`